.operations/gen-html.js

```diff
@@ -7,6 +7,7 @@ const { readdir, readFile, writeFile } = require('graceful-fs');
 const imagemin = require('imagemin');
```

Comment:
@rluvaton Can you kindly remind me what is this file about? Is this new one or an existing file?

Reply:
This is an existing file and I think it's about generating GitHub Pages, but I'm not sure 🤔

Reply:
@BrunoScheufler @js-kyle Do you remember what is the purpose of this file? Is it needed anymore?

Reply:
Do you want to wait for them to answer or can we merge now?